src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenApplicant2Approve.java

```diff
@@ -52,10 +58,16 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             return AboutToStartOrSubmitResponse.<CaseData, State>builder()
                 .data(data)
                 .errors(validationErrors)
-                .state(Draft)
+                .state(AwaitingApplicant2Response)
```

Comment:
maybe to avoid this type of problem it could be `details.getState()` just to keep it as it currently is, but it's good either way